functions/stats.ts

```
@@ -0,0 +1,298 @@
+      hash: game.mainhash,
+    },
+    game: biketagOpts.game,
+  })
+
+  // Get Players data
+  const playersResponse = await biketag.getPlayers(biketagPayload as getPlayersPayload, {
+    source: 'imgur',
+  })
+
+  // Player(s) with most tags in one day (total, time)
+  const playersWithMostTagsInOneDayData: {} = getPlayersWithHighestNumberTagsPerDayData(playersResponse.data)
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)